ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCUI-nonprod.json

```diff
@@ -1,21 +0,0 @@
-[
```

Comment:
Sorry this might have been discussed already, but can I just check why this file is removed?

Reply:
@sampankumar i think it got lost when updating with master, restored and renamed now.